modules/python/src2/hdr_parser.py

```diff
@@ -699,7 +707,7 @@ def parse_stmt(self, stmt, end_token, mat="Mat", docstring=""):
                     decl[1] = ": " + ", ".join([self.get_dotted_name(b).replace(".","::") for b in bases])
                 return stmt_type, classname, True, decl
 
-            if stmt.startswith("class") or stmt.startswith("struct"):
+            if (stmt.startswith("class") or stmt.startswith("struct")) and not stmt.startswith('enum'):
```

Comment:
It's necessary to handle `enum class` there

Reply:
Maybe it may help: https://github.com/opencv/opencv/pull/13471

Reply:
Great, I'll try it

Reply:
Fixed